roles/openshift_node_bootstrap_configmap/defaults/main.yml

```diff
@@ -0,0 +1,27 @@
+---
+openshift_node_bootstrap_confimap_groups:
```

Comment:
Typo, configmap.

Also, if you're going to rename `openshift_master_node_configs` then you should remove it from the master role.

Reply:
The typo was my fault.

Removing from master was definitely a merge issue.

Reply:
Fixed.